vscode-dotnet-runtime-library/src/Utils/IssueReporter.ts

```diff
@@ -0,0 +1,21 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+import { IIssueContext } from './IIssueContext';
+
+const issuesUrl = `https://github.com/dotnet/vscode-dotnet-runtime/issues`;
+
+export function formatIssueUrl(error: Error, context: IIssueContext): string {
+    context.logger.dispose(); // Ensure log file is up to date
+    const issueBody = `<!-- IMPORTANT: Please be sure to remove any private information before submitting and attach the log file located at ${ context.logger.getFileLocation() }. -->
+
+**Repro steps:**
+
+1.
+
+**Error:** ${ error.stack }`;

```

Comment:
You also want the error message don't you?

Reply:
I guess with the error message you'll have to do some privacy work to strip out user identifiable information. We do that [here](https://github.com/dotnet/aspnetcore-tooling/blob/201a91955f88000037f3fc72a58eb5aee941ec18/src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/Diagnostics/ReportIssueCreator.ts#L144) in Razor's tooling

Reply:
In my experimentation it looked like the error message always shows up in the stack trace, so it would be redundant. Is that not always the case?

Reply:
Not entirely sure. I think if an error is manually constructed (imagine VSCode internals throwing for unsupported scenarios) it might be different?